app/test_main.py

```diff
@@ -2,0 +1,21 @@
+import pytest
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "mock_internet,mock_url,expected",
+    [
+        (True, True, "Accessible"),
+        (False, True, "Not accessible"),
+        (True, False, "Not accessible"),
+        (False, False, "Not accessible"),
+    ]
+)
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(mock_internet_con: mock, mock_valid_url: mock, mock_url: mock, mock_internet: mock, expected) -> None:
```

Comment:
The parameters `mock_internet_con` and `mock_valid_url` are used for mocking, but their order in the function signature does not match the order of the `@mock.patch` decorators. The first decorator should correspond to the first parameter in the function signature. Swap the order of `mock_internet_con` and `mock_valid_url` in the function signature to match the decorators.